src/main/java/bio/terra/service/dataset/BigQueryPartitionConfigV1.java

```diff
@@ -14,12 +14,12 @@
     // we want or need to change the settings we collect about BQ
     // partitioning in the future, we'll still be able to distinguish
     // the "old" style of config stored in the DB.
-    @JsonProperty private final long version;
-    @JsonProperty private final Mode mode;
-    @JsonProperty private final String columnName;
-    @JsonProperty private final Long intMin;
-    @JsonProperty private final Long intMax;
-    @JsonProperty private final Long intInterval;
+    @JsonProperty private long version;
+    @JsonProperty private Mode mode;
+    @JsonProperty private String columnName;
+    @JsonProperty private Long intMin;
+    @JsonProperty private Long intMax;
+    @JsonProperty private Long intInterval;
```

Comment:
The members can only be `final` if they are filled in by the constructor. Since we have to use a default constructor for Jackson to be able to deserialize this class, we couldn't make them final anymore.